packages/wrangler/src/dev/local.tsx

```diff
@@ -174,9 +174,13 @@ function useLocalWorker(props: LocalProps) {
 					userWorkerInnerUrlOverrides: {
 						protocol: props.localProtocol,
 						hostname: props.localUpstream,
+						port: props.localUpstream ? "" : undefined, // `localUpstream` was essentially `host`, not `hostname`, so if it was set delete the `port`
```

Comment:
What does an empty string mean for a port?

Reply:
```ts
url = new URL('http://localhost:8787');
console.log(url); // http://localhost:8787
url.port = "";
console.log(url); // http://localhost
```